docs/TemplateDesignGuidance.md

```diff
@@ -153,7 +153,7 @@ Each of the templates in [`example/plugins/templateSelector/templates`](https://
 
 You can update the `isVisible` field to `false` in your bot code based on the `id` of the ActionSet. For example, please refer to the `sendUpdateCard(context: TurnContext)` method in the sample [bot code].
 
-[bot code]: https://github.com/retaildevcrews/wd-bot/blob/arrahm/refactor/example/bot/bot.ts
+[bot code]: https://github.com/retaildevcrews/AdaptiveCardTransformerExampleBot/blob/arrahm/refactor/example/bot/bot.ts
```

Comment:
Got a 404 on this URL. Do we want it to point to main? 

https://github.com/retaildevcrews/AdaptiveCardTransformerExampleBot/tree/main/src/plugins/templateSelector/templates